examples/e2e/search/orderBy.test.js

```diff
@@ -0,0 +1,148 @@
+import { goToSearchPage, checkIfElementExists } from '@vtex/test-tools/cypress'
+
+describe('OrderBy filter', () => {
+  /*
+    Constants
+  */
+  const PRICE_REGEX = /\d{1,3}(?:[.,]\d{3})*(?:[.,]\d{2})/g
+
+  const formatPrice = (price) => {
+    if (!price) return
+
+    return parseFloat(price.replace(',', '.'))
+  }
+
+  before(() => {
+    cy.visit('/')
+    goToSearchPage({ random: true })
+  })
+
+  it('tests OrderByPriceDESC option', () => {
+    cy.getById('refineSearch').select('OrderByPriceDESC')
+
+    cy.url().should('include', 'orderBy=OrderByPriceDESC')
+
+    if (checkIfElementExists('offerPrice')) {
```

Comment:
Why checking if it exists beforehand? I think it's ok to assume that it exists, because if it doesn’t exists, the test will fail with an intuitive error message.

Reply:
It's common to have a category page with all the products out of stock (at least at Marin Brail). The idea is not to fail in this scenario. maybe it's not the default scenario for other stores. What do you think?

Reply:
The test asserts on the different category pages

Reply:
I see, that makes more sense to me now. But even though, there is still the scenario where the  `offerPrice` element _should_ exist but is missing, making it a false positive.

If this is a behavior that can vary from account to account, I suggest making the conditional based on that, [like here](https://github.com/vtex/checkout-ui-tests/blob/master/tests/shipping/models/Delivery%20-%20Boleto.model.js#L27).

Reply:
hmmm good point, what do you think about just removing the `if` of the example and keeping that way in Marin? I think each store can make better decisions on how to test it depending on the context(Marin is specific)!

Reply:
@wdsrocha

Reply:
Sure! That looks good to me.